templates/cost-flow-ortools/main.py

```diff
@@ -0,0 +1,281 @@
+"""
+Template to solve a cost flow problem with Google OR-Tools.
+"""
+
+import argparse
+import json
+import sys
+import time
+from typing import Any
+
+from ortools.graph.python import min_cost_flow
+
+STATUS = {
+    min_cost_flow.SimpleMinCostFlow.OPTIMAL: "optimal",
+    min_cost_flow.SimpleMinCostFlow.BAD_COST_RANGE: "bad cost range",
+    min_cost_flow.SimpleMinCostFlow.BAD_RESULT: "bad result",
+    min_cost_flow.SimpleMinCostFlow.FEASIBLE: "feasible",
+    min_cost_flow.SimpleMinCostFlow.INFEASIBLE: "infeasible",
```

Comment:
Is it actually possible to have an infeasible result or any other result that has no solution values? If so, it would be great if we would test for it so that the app does not crash in those situations. We had/have this with some mip templates.

Reply:
One thing I found: if I add a required skill that is not available, I get a segfault. It would be good, if we could catch such situations.